packages/react-components/src/components/utils/filtrer-tree/checkbox-component.tsx

```diff
@@ -30,8 +30,8 @@ export class CheckboxComponent extends React.Component<CheckboxProps> {
     };
 
     render(): JSX.Element {
-        return <span style={{padding: 5}} onClick={this.handleClick}>
+        return <div style={{paddingRight: 5}} onClick={this.handleClick}>
```

Comment:
Use inline as in: `return <div style={{ paddingRight: 5, display: 'inline' }} onClick={this.handleClick}>`